tasks/components/watchdog/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+# Check if the variable is not blank
+if [[ ! -z "$CONFIG_DS_WATCHDOG_DEVICE" ]]; then
+    # Write the configuration to /etc/watchdog.conf
+    cat > /etc/watchdog.conf << EOF
+watchdog-device = $CONFIG_DS_WATCHDOG_DEVICE
+watchdog-timeout = $CONFIG_DS_WATCHDOG_TIMEOUT
```

Comment:
Good point.  Because we've seen clock inaccuracies on the watchdog domain I'm going to set this to /4:

```
INTERVAL=$((CONFIG_DS_WATCHDOG_TIMEOUT/4))
if ((INTERVAL < 1)); then
    INTERVAL=1
fi
```